docs/advanced/advanced_troubleshooting.md

```diff
@@ -0,0 +1,151 @@
+    |-vda2   252:2    0   1G  0 part  /boot
+    `-vda3   252:3    0  30G  0 part  /sysroot
+    </pre>
+
+    To display device attachment information, together with SCSI ID information, use the `sudo lsblk -S` command.
+
+    <pre>
+    NAME HCTL       TYPE VENDOR   MODEL             REV TRAN
+    sda  3:0:0:0    disk IBM      2145             0000 iscsi
+    sdb  2:0:0:0    disk IBM      2145             0000 iscsi
+    </pre>
+
+-   **Check for multipath daemon availability (FC and iSCSI)**
+
+    Check for multipath daemon availability, using the `systemctl status multipathd` command.
+
+    <pre>
+    multipathd.service - Device-Mapper Multipath Device Controller
+        Loaded: loaded (/usr/lib/systemd/system/multipathd.service; enabled; vendor preset: enabled)
+        Active: active (running) since Mon 2020-03-09 16:28:37 UTC; 22min ago
+      Main PID: 1235 (multipathd)
+        Status: "up"
+        Tasks: 7
+        Memory: 14.1M
+            CPU: 131ms
+        CGroup: /system.slice/multipathd.service
+                └─1235 /sbin/multipathd -d -s
+    </pre>
+
+-   **Check for iSCSI daemon availability**
+
+    Check for iSCSI daemon availability, using the `systemctl status iscsid` command.
+
+    <pre>
+    iscsid.service - Open-iSCSI
+      Loaded: loaded (/usr/lib/systemd/system/iscsid.service; enabled; vendor preset: disabled)
+      Active: active (running) since Mon 2020-03-09 16:28:37 UTC; 22min ago
+        Docs: man:iscsid(8)
+              man:iscsiadm(8)
+    Main PID: 1440 (iscsid)
+      Status: "Ready to process requests"
+        Tasks: 1 (limit: 26213)
+      Memory: 4.7M
+          CPU: 27ms
+      CGroup: /system.slice/iscsid.service
+              └─1440 /usr/sbin/iscsid -f
+    </pre>
+
+## General troubleshooting
+
+Use the following command for general troubleshooting:
+
+    kubectl get -n <namespace>  csidriver,sa,clusterrole,clusterrolebinding,statefulset,pod,daemonset | grep ibm-block-csi
+
+For example:
+
+    $> kubectl get -n <namespace>  csidriver,sa,clusterrole,clusterrolebinding,statefulset,pod,daemonset | grep ibm-block-csi
+    csidriver.storage.k8s.io/ibm-block-csi-driver 7d
+
+    serviceaccount/ibm-block-csi-controller-sa 1 2h
+    serviceaccount/ibm-block-csi-node-sa 1 2h
+    serviceaccount/ibm-block-csi-operator 1 2h
+
+    clusterrole.rbac.authorization.k8s.io/ibm-block-csi-external-attacher-clusterrole 2h
+    clusterrole.rbac.authorization.k8s.io/ibm-block-csi-external-provisioner-clusterrole 2h
+    clusterrole.rbac.authorization.k8s.io/ibm-block-csi-operator 2h
+
+    clusterrolebinding.rbac.authorization.k8s.io/ibm-block-csi-external-attacher-clusterrolebinding 2h
+    clusterrolebinding.rbac.authorization.k8s.io/ibm-block-csi-external-provisioner-clusterrolebinding 2h
+    clusterrolebinding.rbac.authorization.k8s.io/ibm-block-csi-operator 2h
+
+
+    statefulset.apps/ibm-block-csi-controller 1 1 2h
+    pod/ibm-block-csi-controller-0 6/6 Running 0 2h
```

Comment:
DONE

Reply:
I meant that we don't want to maintain this output anywhere

Reply:
I'm not even sure what is the purpose of this section.
IMO it is very vague, and I would remove it

Reply:
@oriyarde  Here is the entire section - please specify if I should remove the entire example or just parts of it. Thanks

> ## General troubleshooting
> 
> Use the following command for general troubleshooting:
> 
>     kubectl get -n <namespace>  csidriver,sa,clusterrole,clusterrolebinding,statefulset,pod,daemonset | grep ibm-block-csi
> 
> For example:
> 
>     $> kubectl get -n <namespace>  csidriver,sa,clusterrole,clusterrolebinding,statefulset,pod,daemonset | grep ibm-block-csi
>     csidriver.storage.k8s.io/ibm-block-csi-driver 7d
> 
>     serviceaccount/ibm-block-csi-controller-sa 1 2h
>     serviceaccount/ibm-block-csi-node-sa 1 2h
>     serviceaccount/ibm-block-csi-operator 1 2h
> 
>     clusterrole.rbac.authorization.k8s.io/ibm-block-csi-external-attacher-clusterrole 2h
>     clusterrole.rbac.authorization.k8s.io/ibm-block-csi-external-provisioner-clusterrole 2h
>     clusterrole.rbac.authorization.k8s.io/ibm-block-csi-operator 2h
> 
>     clusterrolebinding.rbac.authorization.k8s.io/ibm-block-csi-external-attacher-clusterrolebinding 2h
>     clusterrolebinding.rbac.authorization.k8s.io/ibm-block-csi-external-provisioner-clusterrolebinding 2h
>     clusterrolebinding.rbac.authorization.k8s.io/ibm-block-csi-operator 2h
> 
> 
>     statefulset.apps/ibm-block-csi-controller 1 1 2h
>     pod/ibm-block-csi-controller-0 7/7 Running 0 2h
>     pod/ibm-block-csi-node-nbtsg 3/3 Running 0 2h
>     pod/ibm-block-csi-node-wd5tm 3/3 Running 0 2h
>     pod/ibm-block-csi-operator-7684549698-hzmfh 1/1 Running 0 2h
> 
>     daemonset.extensions/ibm-block-csi-node 2 2 2 2 2 <none> 2h

Reply:
IMO we should not have partial outputs, so I'm in favor of removing the entire output.
and since I don't understand this section's purpose, I'm in favor of removing this entire section

Reply:
@lisyukanna  @barakda can we just remove the entire output? is there any necessity to keep it in the advanced documentation?

Reply:
Agree with @oriyarde's suggestion, I think it is better to remove the entire output.
It is clear as it is and no future maintenance is required

Reply:
DONE

Reply:
thanks! much better.
could you also remove it from [here](https://github.com/IBM/ibm-block-csi-driver/blob/b0e5c1dffeb944e6d96edccdd74ace0560ed0ee5/USAGE-DETAILS.md#troubleshooting)? or should we do it in a separate PR/ticket?

Reply:
I'm not sure what that section is or where it's from (is it being used?)
I think update it separately. thanks

Reply:
I sure hope it's not being used.
got it, np